### PR TITLE
Global : Update the wording 'public' and use the new room creation UI

### PR DESCRIPTION
**Pull request by Yoshiin**
Resolve dinsic-pim/tchap-web#119

### CODE REVIEW

**Comment by Yoshiin** [comment]
Here is the new UI :
TchapRoomCreation

File: src/i18n/strings/fr.json [hunk 1]
```diff
@@ -1938,7 +1938,7 @@
     "Share": "Partager",
     "Try again later, or ask a room admin to check if you have access.": "Réessayez plus tard ou demandez à un administrateur du salon si vous y avez accès.",
     "Are you sure you want to remove this room from the rooms directory?": "Voulez-vous vraiment retirer ce salon des salons publics ?",
-    "Remove this room from the rooms directory": "Retirer ce salon de la liste des salons publics",
+    "Remove this room from the rooms directory": "Retirer ce salon de la liste des salons forum",
```
Comment (giomfo) [inline src/i18n/strings/fr.json:1941]:
```suggestion
    "Remove this room from the rooms directory": "Retirer ce salon de la liste des salons forums",
```

**Review by giomfo — COMMENTED** [review]
Commented, one more comment is about the avatar: you update the avatar border according to the selected option (which is good), I think you should add the padlock or the forum icon too

File: src/i18n/strings/fr.json [hunk 2]
```diff
@@ -1958,5 +1958,15 @@
     "Stay connected from at least one other device": "Restez connecté depuis au moins un autre appareil",
     "Export your encryption keys": "Exporter vos clés de chiffrement",
     "Find out more": "En savoir plus",
-    "Export keys and log-out": "Exporter les clés et se déconnecter"
+    "Export keys and log-out": "Exporter les clés et se déconnecter",
+    "Private room": "Salon privé",
+    "Accessible to all gouv.fr users by invitation from an administrator.": "Accessible à tous les utilisateurs gouv.fr sur invitation d’un administrateur.",
```
[the remark this answers is not in the document]
Comment (Yoshiin) [reply src/i18n/strings/fr.json:1963]: Suggested : "Accessible à tous les utilisateurs sur invitation d’un administrateur."

File: src/i18n/strings/fr.json [hunk 3]
```diff
@@ -1958,5 +1958,15 @@
...
+    "Private room opened to externals": "Salon privé ouvert aux externes",
+    "Accessible to all gouv.fr users and to external guests by invitation of an administrator.": "Accessible à tous les utilisateurs gouv.fr et aux invités externes sur invitation d’un administrateur.",
```
[the remark this answers is not in the document]
Comment (Yoshiin) [reply src/i18n/strings/fr.json:1965]: Suggested : "Accessible à tous les utilisateurs et aux invités externes sur invitation d’un administrateur."

File: src/i18n/strings/fr.json [hunk 4]
```diff
@@ -1958,5 +1958,15 @@
...
+    "Forum room": "Salon forum",
+    "Accessible to all gouv.fr users from the directory or from a shared link.": "Accessible à tous les utilisateurs gouv.fr à partir de l’annuaire ou d’un lien partagé.",
```
[the remark this answers is not in the document]
Comment (Yoshiin) [reply src/i18n/strings/fr.json:1967]: Suggested : "Accessible à tous les utilisateurs à partir de la liste des forums ou d’un lien partagé."

File: src/components/structures/RoomDirectory.js [hunk 5]
```diff
@@ -536,7 +536,7 @@ module.exports = React.createClass({
                 hasCancel={true}
                 onFinished={this.props.onFinished}
                 headerButton={createRoomButton}
-                title={_t("Room directory")}
+                title={_t("Forum room directory")}
```
Comment (giomfo) [inline src/components/structures/RoomDirectory.js:539]: I would rename this "Forum directory" = "Annuaire/Répertoire des forums"

File: src/components/views/settings/tabs/room/GeneralRoomSettingsTab.js [hunk 6]
```diff
@@ -63,7 +63,7 @@ export default class GeneralRoomSettingsTab extends React.Component {
 
         Modal.createTrackedDialog('Remove this room from the rooms directory', '', QuestionDialog, {
             title: _t('Remove this room from the rooms directory'),
-            description: ( _t('This action is irreversible.') + " " + _t('Are you sure you want to remove this room from the rooms directory?')),
+            description: ( _t('This action is irreversible.') + " " + _t('Are you sure you want to remove this room from the forum rooms directory?')),
```
Comment (giomfo) [inline src/components/views/settings/tabs/room/GeneralRoomSettingsTab.js:66]: I would use "forums directory" here

File: src/i18n/strings/en_EN.json [hunk 7]
```diff
@@ -1693,5 +1693,12 @@
     "Stay connected from at least one other device": "Stay connected from at least one other device",
     "Export your encryption keys": "Export your encryption keys",
     "Find out more": "Find out more",
-    "Export keys and log-out": "Export keys and log-out"
+    "Private room": "Private room",
+    "Accessible to all users by invitation from an administrator.": "Accessible to all users by invitation from an administrator.",
+    "Private room opened to externals": "Private room opened to externals",
+    "Accessible to all users and to external guests by invitation of an administrator.": "Accessible to all users and to external guests by invitation of an administrator.",
+    "Forum room": "Salon forum",
```
Comment (giomfo) [inline src/i18n/strings/en_EN.json:1700]: French in English string...

File: src/i18n/strings/en_EN.json [hunk 8]
```diff
@@ -1693,5 +1693,12 @@
...
+    "Accessible to all users from the forum directory or from a shared link.": "Accessible à tous les utilisateurs à partir de la liste des forums ou d’un lien partagé.",
```
Comment (giomfo) [inline src/i18n/strings/en_EN.json:1701]: French in English strings...

File: src/i18n/strings/fr.json [hunk 9]
```diff
@@ -1958,5 +1958,15 @@
...
+    "Private room": "Salon privé",
+    "Accessible to all users by invitation from an administrator.": "Accessible à tous les utilisateurs sur invitation d’un administrateur.",
+    "Private room opened to externals": "Salon privé ouvert aux externes",
+    "Accessible to all users and to external guests by invitation of an administrator.": "Accessible à tous les utilisateurs et aux invités externes sur invitation d’un administrateur.",
+    "Forum room": "Salon forum",
+    "Accessible to all users from the forum directory or from a shared link.": "Accessible à tous les utilisateurs à partir de la liste des forums ou d’un lien partagé.",
+    "Create a private room opened to externals": "Créer un salon privé ouvert aux externes",
+    "Create a forum room": "Créer un salon forum",
+    "Forum room directory": "Répertoire des salons forum",
+    "Are you sure you want to remove this room from the forum rooms directory?": "Voulez-vous vraiment retirer ce salon des salons forum ?"
```
Comment (giomfo) [inline src/i18n/strings/fr.json:1971]: "... retirer ce salon des forums ?"

**Review by giomfo — COMMENTED** [review]
SGTM, I just suggested using directly "forum"  in few cases (instead of "forum room")

File: src/i18n/strings/fr.json [hunk 10]
```diff
@@ -1938,7 +1938,7 @@
     "Share": "Partager",
     "Try again later, or ask a room admin to check if you have access.": "Réessayez plus tard ou demandez à un administrateur du salon si vous y avez accès.",
     "Are you sure you want to remove this room from the rooms directory?": "Voulez-vous vraiment retirer ce salon des salons publics ?",
```
Comment (giomfo) [inline src/i18n/strings/fr.json:1940]: Sorry we missed this occurrence "salons publics"

Comment (giomfo) [reply src/i18n/strings/fr.json:1940]: this string is not used

**Review by giomfo — APPROVED** [review]

**Pull request merged**